tools/shared.py

```diff
@@ -2514,6 +2514,18 @@ def minify_wasm_js(js_file, wasm_file, expensive_optimizations, minify_whitespac
           js_file = Building.minify_wasm_imports_and_exports(js_file, wasm_file, minify_whitespace=minify_whitespace, minify_exports=Settings.DECLARE_ASM_MODULE_EXPORTS, debug_info=debug_info)
     return js_file
 
+  WASI_IMPORTS = set([
+    'environ_get',
+    'environ_sizes_get',
+    'fd_write',
+    'fd_close',
+    'fd_read',
+    'fd_seek',
+    'fd_fdstat_get',
+    'fd_sync',
+    'proc_exit',
+  ])
+
```

Comment:
Why move this list?

Reply:
I thought it might be cleaner, and maybe useful from other areas, but I can revert this part. I ended up not needing it here.